{{cookiecutter.project_slug}}/README.md

```diff
@@ -59,19 +59,54 @@ Centre for Advanced Research Computing, University College London
 
 ### Prerequisites
 
-Any tools or versions of languages needed to run code. For example specific Python or Node versions. Minimum hardware requirements also go here.
+<!-- Any tools or versions of languages needed to run code. For example specific Python or Node versions. Minimum hardware requirements also go here. -->
+
+`{{cookiecutter.project_slug}}` requires Python {{cookiecutter.min_python_version}}{% if cookiecutter.min_python_version != cookiecutter.max_python_version %}&ndash;{{cookiecutter.max_python_version}}{% endif %}.
 
 ### Installation
 
-How to build or install the application.
+<!-- How to build or install the application. -->
+
+To install the latest development version of `{{cookiecutter.project_slug}}` using `pip` run
+
+```sh
+pip install git+https://github.com/{{cookiecutter.github_username}}/{{cookiecutter.project_slug}}.git
+```
+
+Alternatively create a local clone of the repository with
```

Comment:
Do we want to get into the business of setting up a mamba(`conda`) environment?
It's one of our [recommended dev tools](http://github-pages.arc.ucl.ac.uk/python-tooling/pages/dev.html#virtual-environments).

Reply:
Good point - have had a bit of text say we recommend installing in a virtual environment using a tool like Mamba or Conda with links to their docs - haven't included specific instructions of setting up an environment but hopefully links will be sufficient breadcrumb if people are not familiar how to do this already.